app.js

```diff
@@ -13,5 +13,392 @@ menuIcon.onclick = () => {
+
+  const popupTitle = createElement('h3', 'popupTitle');
+  popupTitle.innerHTML = projectCardInfo.title;
+  popupHeader.appendChild(popupTitle);
+
+  const popupLanguages = createList(projectCardInfo.technologies, 'popupLanguages');
+  popupHeader.appendChild(popupLanguages);
+
+  const popupBody = createElement('div', 'popupBody');
+  docFragment.appendChild(popupBody);
+
+  const popupImgContainer = createElement('div', 'popupImgContainer');
+  popupBody.appendChild(popupImgContainer);
+
+  const popupImg = createElement('img', 'popupImg');
+  popupImg.src = projectCardInfo.image;
+  popupImgContainer.appendChild(popupImg);
+
+  const popupTextContainer = createElement('div', 'popupTextContainer');
+  popupBody.appendChild(popupTextContainer);
+
+  const popupDescription = createElement('p', 'popupDescription');
+  popupDescription.innerText = projectCardInfo.description;
+  popupTextContainer.appendChild(popupDescription);
+
+  const popupLinkContainer = createElement('div', 'popupLinkContainer');
+  popupTextContainer.appendChild(popupLinkContainer);
+
+  const liveLink = createElement('a', 'liveLink');
+  liveLink.innerHTML = 'See live&nbsp;&nbsp;<i class="uil uil-export"></i>';
+  liveLink.href = projectCardInfo.liveLink;
+  popupLinkContainer.appendChild(liveLink);
+
+  const sourceLink = createElement('a', 'sourceLink');
+  sourceLink.innerHTML = 'See source&nbsp;&nbsp;<i class="uil uil-github"></i>';
+  sourceLink.href = projectCardInfo.sourceLink;
+  popupLinkContainer.appendChild(sourceLink);
+
+  projectPopup.appendChild(docFragment);
+  popupContainer.appendChild(projectPopup);
+  body.appendChild(popupContainer);
+
+  document.querySelector('#bar').style.filter = 'blur(10px)';
+
+  document.querySelector('#firstsection').style.filter = 'blur(10px)';
+  document.querySelector('#about-me').style.filter = 'blur(10px)';
+
+  const closeBtn = document.getElementById('closeBtn');
+  closeBtn.addEventListener('click', () => {
+    body.removeChild(popupContainer);
+    document.querySelector('#bar').style.filter = 'blur(0)';
+
+    document.querySelector('#firstsection').style.filter = 'blur(0)';
+    document.querySelector('#about-me').style.filter = 'blur(0)';
+  });
+}
+
+const projectsBtn = document.querySelectorAll('.project-btn');
+projectsBtn.forEach((btn) => btn.addEventListener('click', projectModal));
+const projectsBtn2 = document.querySelectorAll('.work1Link');
+projectsBtn2.forEach((btn) => btn.addEventListener('click', projectModal));
+
+//= ========form validation=========================
+const form = document.querySelector('.contactForm');
+const email = document.querySelector('#email');
+const errorMessage = document.querySelector('.errorMessage');
+
+form.addEventListener('submit', (e) => {
+  if (email.value !== email.value.toLowerCase()) {
+    e.preventDefault();
+    errorMessage.textContent = 'Your email input must be in lowercases!';
+  } else {
+    errorMessage.textContent = '';
+  }
+});
```

Comment:
- [ ] `README.md` file: To make your Documentation be more professional and as required for every project, I think it would be better if you add all the contributor authors in your README.